src/99-appendices/05-units.md

```diff
@@ -1,39 +1,78 @@
-| newton         | N           | force, weight                              |
-| pascal         | Pa          | pressure, stress                           |
-| joule          | J           | energy, work, heat                         |
-| watt           | W           | power, radiant flux                        |
-| coulomb        | C           | electric charge or quantity of electricity |
-| volt           | V           | voltage (electrical potential), emf        |
-| farad          | F           | capacitance                                |
-| ohm            | Ω           | resistance, impedance, reactance           |
-| siemens        | S           | electrical conductance                     |
-| weber          | Wb          | magnetic flux                              |
-| tesla          | T           | magnetic flux density                      |
-| henry          | H           | inductance                                 |
-| degree Celsius | °C          | temperature relative to 273.15 K           |
-| lumen          | lm          | luminous flux                              |
-| lux            | lx          | illuminance                                |
-| becquerel      | Bq          | radioactivity (decays per unit time)       |
-| gray           | Gy          | absorbed dose (of ionizing radiation)      |
-| sievert        | Sv          | equivalent dose (of ionizing radiation)    |
-| katal          | kat         | catalytic activity                         |
+The [CMIXF-12](https://people.csail.mit.edu/jaffer/MIXF/CMIXF-12) convention
+for encoding units is REQUIRED to achieve maximum portability and limited
+variability of representation.
+If a CMIXF-12 representation of a unit is not possible, the unit MUST be declared
+as custom units and defined in an accompanying JSON file, as described in the
+[units section](../02-common-principles.md#units).
+Earlier versions of the BIDS standard listed the following Unicode symbols, and
+these are still included for backwards compatibility:
+
+1.  `U+00B5` (µ)
+1.  `U+00B0` (°)
+1.  `U+2126` (Ω).
+
+It is REQUIRED that units be CMIXF-12 compliant or among these three Unicode
+characters.
+Please take care not to confuse the accepted Unicode character
+[`U+00B5`](https://codepoints.net/U+00B5) (µ) with the disallowed Unicode
+character [`U+03BC`](https://codepoints.net/U+03BC) (μ):
+They look equal in appearance but are different to the computer.
+
+Units MUST consist of the `unit symbol` with an optionally accompanying
+`prefix symbol` (see table below). Appropriate upper- or lower- casing MUST
+be applied as declared by CMIXF-12.
+
+For cases that are unspecified by this appendix, or the
+[units section](../02-common-principles.md#units), the
+[CMIXF-12](https://people.csail.mit.edu/jaffer/MIXF/CMIXF-12) convention
+applies.
+You can use the [cmixf Python package](https://github.com/sensein/cmixf) to
+check whether your formatting is compliant.
```

Comment:
is there a corresponding issue on the BIDS-validator? Ideally, the validator should tell you instead of having to use another tool. Also works for people who are not familiar with Python.

Reply:
Not yet, but @rwblair is aware of it and I'll open an issue as soon as this PR is in to continue tracking this challenge.

Reply:
now there is an issue: https://github.com/bids-standard/bids-validator/issues/969